pipeline/tasks/protoc_tasks.py

```diff
@@ -218,8 +259,8 @@ def execute(self, src_proto_path, import_proto_path, output_dir,
         #   - it doesn't have to
         #   - and multiple invocation will overwrite the desc_out_file
         self.exec_command(
-            _protoc_header_params(
-                header_proto_path, toolkit_path) +
+            ['protoc'] +
```

Comment:
Think about users who are only interested in Ruby (or think that remote environment which only takes care of Ruby). Considering those environment, requiring both `protoc` and `grpc_tools_ruby_protoc` doesn't sound great, because the latter has the same feature already.